### PR TITLE
Crypter v chiffrer

### PR DESCRIPTION
**Pull request by Kaboum**
The word "crypter" doesn't really exists. "Chiffré" is better. :)

### CODE REVIEW

**Comment by baldurmen** [comment]
"Chiffrement" is indeed the right work in French, but then again, nobody uses that were I live (Quebec) where "encryption" & "cryptage" are common terms. I can't talk for France though.

IMHO we should use words people understand more than dictionary correct words.

**Comment by micah** [comment]
Louis-Philippe Véronneau <notifications@github.com> writes:

> "Chiffrement" is indeed the right work in French, but then again, nobody uses that were I live (Quebec) where "encryption" & "cryptage" are common terms. I can't talk for France though.
>
> IMHO we should use words people understand more than dictionary correct words.

If its what people use in France, then we should meet for dîner and
figure out what to do ;)

Seriously though, maybe if France people wont understand it, and
Québécoise wouldn't understand the other way, what about putting both
like 'cryptage/chiffrement'? Its a little awkward, but its better than
having one not understand something

**Comment by baldurmen** [comment]
Meh, I think the cryptage/chiffrement' is the most awkward solution.

If people in France _do_ use 'chiffrement', I don't really mind since
they are 10 times more than us.

I just don't want to use dictionary French just for the sake of it.

**Comment by anarcat** [comment]
> The word "crypter" doesn't really exists. "Chiffré" is better. :)

`{{citation needed}}` it's useless to just disparage existing work without backing it up with some research or some facts. in fact, i feel it's offensive for the people that did that work in the first place.

"cryptage" actually exists:

http://granddictionnaire.com/ficheOqlf.aspx?Id_Fiche=8387607
https://fr.wiktionary.org/wiki/cryptage
https://fr.wikipedia.org/wiki/Cryptage (redirects to chiffrement)

> If people in France _do_ use 'chiffrement', I don't really mind since
> they are 10 times more than us.

I do not care if there are 10 times more of "you". First, it's inaccurate: there's 67M people in france, and 8.2M in Québec. So even if you ignore french speakers outside of quebec (which would be a gross mischaracterization of Francophonie), it's still inaccurate (the number you were looking for is "8" here). Still according to wikipedia, there are 274M people speaking french across the world. Saying that you are more than us is simply offensive and shows a serious misunderstanding of history and colonial power dynamics.

I suggest you bring your arguments more delicately the next time. I think what you meant to say is "there are 4 times more of you so maybe i'll start listening to the colonies for a change".

But anyways... the point here is to find the *correct* meaning that is accurate in the maximum number of places. This is a classic localization problem:

https://fr.wikipedia.org/wiki/Chiffrement#Terminologie

One official government body (OQLF in Québec) thinks that both words are fine to use, but with an emphasis on saying that "décryptage" means "breaking encryption without the key", impying that "cryptage" is stronger than "chiffrement". Indeed, "crypter" is used to talk about TV encryption in France which is hilariously considered "unbreakable" according to the vocabulary used by the ANSSI.

in short, i don't care much: i agree that "cryptage/chiffrement" (with the slash) is a bad solution. just choose a solution and stick with it. 

I prefer "encryption" for the name, but that seems to be an "anglicisme". "chiffrement" is also acceptable, but is more ambiguous as a verb: "chiffrer" can also mean "count" ("le ministre a chiffré les dépenses..."), so I would prefer "crypter" and "décrypter" there.

good luck!

**Comment by jcharaoui** [comment]
For what it's worth, the Enigmail project translates "Encrypt" and "Decrypt" as "Chiffrer" and "Déchiffrer". See http://beta.babelzilla.org/projects/p/Enigmail/resource/enigmaildtd/l/fr/view/

**Comment by baldurmen** [comment]
@Kaboum I'm the one who translated these pages, so don't worry, I do not feel offended whatsoever. 

To me this is really a non-issue since a lot of work is still to be done on the help pages: lot of them are not yet translated in French and in my honest opinion we should work on translating them first.

Once this is done, we should effectively to to use the same vocabulary everywhere (be it "encryption" or "chiffrement"), but I think we are not there yet.

**Comment by anarcat** [comment]
oh, sorry for the confusion - i had @baldurmen  and @Kaboum confused and assumed they were from france.

even so, my response was a bit too harsh. my apologies.

**Comment by elijh** [comment]
is there consensus on this? should this be merged?

**Comment by baldurmen** [comment]
@elijh I think there was no consensus on this :D. IMHO it should thus not be merged.

**Comment by manhack** [comment]
Hi,

let me intervene in english, for anarcat and the other non-french ppl (as
everybody in here understand english) : from a french-speaking AND security
point of view, we tend to discard and reject the "crypter/cryptage" wording
as it is mainly used by people who don't understand encryption (and
security) issues.. although our goal is precisely to help them (and the
ppl) properly understand how this works, and that's why we usually prefer
"chiffrer/chiffrement"

bien cordialement,
jm @manhack <http://twitter.com/manhack>, envoyé spécial sur les internets
(envoyé depuis les "nuages" -placés sous surveillances)

2016-04-21 1:23 GMT+02:00 anarcat <notifications@github.com>:

**Comment by anarcat** [comment]
my comments shouldn't be construed as blocking consensus. a merge would be good only if to improve consistency.

i think a standard translation lexicon would be in order...

**Comment by baldurmen** [comment]
@anarcat I'm not in favor of merging (yet) either. I like the lexicon idea (see issue #322).

**Comment by baldurmen** [comment]
I'm leaving this here because I find it quite funny: https://chiffrer.info/

**Comment by just1602** [comment]
@baldurmen, it's funny, but do you agree with those arguments? If so, we could merge and close this PR! :D

**Comment by jcharaoui** [comment]
Enigmail and [GnuPG](http://sources.debian.net/src/gnupg/1.4.20-6/po/fr.po/#L697) both translate "encryption" as "chiffrement". So even if "crypter" is more widely used in spoken language is some regions (because no one has provided any data on this), for the sake of consistency with the actual tools which are used for encryption, this should be merged.

**Comment by elijh** [comment]
can someone who speaks french and git do a rebase for this PR so it can be merged?
